### PR TITLE
osa - update master config and nits

### PR DESCRIPTION
**Pull request by mjudeikis**
Update master to point to 3.11.
Add conformance as periodic
rollback AAD change

/cc @kwoodson @kargakis @charlesakalugwu

### CODE REVIEW

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml [hunk 1]
```diff
@@ -279,3 +279,56 @@ periodics:
         requests:
           cpu: 10m
     serviceAccountName: ci-operator
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  interval: 24h
+  name: azure-test-conformance
```
Comment (kargakis) [inline ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml]: We need to start validating these names in CI. Or at least document the conventions we follow:
* `periodic-ci-{org}-{repo}-{branch}-{name}` for periodic jobs
eg. periodic-ci-openshift-openshift-azure-master-conformance
* `pull-ci-{org}-{repo}-{branch}-{name}` for PR (presubmit) jobs
eg. pull-ci-openshift-openshift-azure-master-conformance
* `branch-ci-{org}-{repo}-{branch}-{name}` for branch (postsubmit) jobs
eg. branch-ci-openshift-openshift-azure-master-conformance

Comment (mjudeikis) [reply ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml]: agreed. And potentially document all command available in our repos too.

Comment (mjudeikis) [reply ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml]: But this sound as separate story/issue

Comment (kargakis) [reply ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml]: At least follow the convention aka update the name to `periodic-ci-openshift-openshift-azure-conformance`.

**Comment by mjudeikis** [comment]
@kargakis if this is fine, will raise separate story for names documentation

**Pull request merged**